CHANGELOG.md

```diff
@@ -1,3 +1,16 @@
+## 1.14.0 - 21 June 2021
+
+### Added
+
+* Enable Dockerfile formatter to skip instructions that span multiple lines [2976](https://github.com/microsoft/vscode-docker/pull/2976)
+* Add ability to see dangling images in the explorer [2967](https://github.com/microsoft/vscode-docker/pull/2967)
+* Add a setting for Docker CLI executable path [2984](https://github.com/microsoft/vscode-docker/pull/2984), providing a workaround for [issue 2894](https://github.com/microsoft/vscode-docker/issues/2894)
```

Comment:
There seems to be some inconsistency here. All of the links are just the number `1234` but 2984 is `issue 1234`. The older changelog entries also linked to the issues with `#1234`. Can these machine-generated entries be made to follow the old format? 🤔

Reply:
@rcjsuen you are right, I should be using issue references, not PR references. Will fix it, but I also need to abandon this PR and not use the release branch for this. Stay tuned.